src/subcommand/server.rs

```diff
@@ -1441,6 +1451,47 @@ impl Server {
     })
   }
 
+  async fn sat_inscriptions(
+    Extension(index): Extension<Arc<Index>>,
+    Path(sat): Path<u64>,
```

Comment:
```suggestion
    Path(sat): Path<Sat>,
```

Reply:
Do we want Sat? Raph had me take it out of mine. Would also require more complexity for any other alternative clients supporting this endpoint.

Reply:
wrong function disregard.